python/pyiceberg/expressions/visitors.py

```diff
@@ -753,3 +756,61 @@ def inclusive_projection(
     schema: Schema, spec: PartitionSpec, case_sensitive: bool = True
 ) -> Callable[[BooleanExpression], BooleanExpression]:
     return InclusiveProjection(schema, spec, case_sensitive).project
+
+
+class _ColumnNameTranslator(BooleanExpressionVisitor[BooleanExpression]):
+    """Converts the column names with the ones in the actual file
+
+    Args:
+      table_schema (Schema): The schema of the table
```

Comment:
Minor: `table_schema` is no longer passed.

Reply:
I've been looking for a checker for a while, but haven't found one yet 😭